dash/development/component_generator.py

```diff
@@ -22,7 +26,28 @@ class _CombinedFormatter(argparse.ArgumentDefaultsHelpFormatter,
 
 # pylint: disable=too-many-locals
 def generate_components(components_source, project_shortname,
-                        package_info_filename='package.json'):
+                        package_info_filename='package.json',
+                        generate_r_components=False,
+                        rprefix=None):
+    project_shortname = project_shortname.replace('-', '_').rstrip('/\\')
+
+    # import component library module
+    importlib.import_module(project_shortname)
+
+    if rprefix:
+        prefix = rprefix
+    else:
+        s = [
+            x for x in project_shortname.split('_')
+            if x not in ('dash', 'component', 'components')
+        ]
+        prefix = s[-1]
+        print(
+            'Warning: a component prefix was '
```

Comment:
it's not clear from this error message that it refers to R only

Reply:
Right, will revise this message so it's obvious which pipeline is involved here.